.jenkins/pytorch/win-test.sh

```diff
@@ -72,13 +72,13 @@ run_tests() {
           export PYTORCH_COLLECT_COVERAGE=1
           export COVERAGE_RCFILE=$PWD/.coveragerc # coverage config file needed for plug-ins and settings to work
         fi
-        if [[ "${JOB_BASE_NAME}" == *-test1 ]]; then
+        if [[ "${JOB_BASE_NAME}" == *-test1 || ( $NUM_TEST_SHARDS -gt 1 && "${TEST_CONFIG}" == 1 )]]; then
```

Comment:
the `-gt` here feels a bit weird: shouldn't it be `-eq 2`?

Reply:
I was hoping we were able to scale up to more than 2 shards but yes the current implementation does not lend itself that way and this should be -eq 2